src/components/menu/index.css

```diff
@@ -208,6 +208,10 @@ li.desktop-only-link {
         /* width: auto; */
     }
 
+    .submenu-wrapper > ul {
+        left: -8rem !important;
```

Comment:
@Shebuka I probably just went a bit too wild with the `-8rem`. Perhaps if we did `-6rem` it would fit better?